ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10.yaml

```diff
@@ -411,22 +411,8 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-upi
-- as: gcp-ipi-cco-manual-workload-identity-fips-p2-f14
```

Comment:
Recently we are adding many fips jobs on purpose for some urgent testing for all versions, so maybe we can close this PR. Later I will file some new PR to follow the idea to get these configuration combined.